src/blocks/ui/components/timeline.js

```diff
@@ -64,6 +64,7 @@ const StyledTimeline = styled.div`
 			text-align: right;
 			bottom: -4px;
 			padding-bottom: 4px;
+			white-space: nowrap;
```

Comment:
Do! Is that all it was? I thought I'd tried that, but maybe I tried it in the wrong place! Cheers.